communication/socket/send/send-data-on-socket.yml

```diff
@@ -2,7 +2,9 @@ rule:
   meta:
     name: send data on socket
     namespace: communication/socket/send
-    author: moritz.raabe@fireeye.com
+    author:
+      - moritz.raabe@fireeye.com
+      - joakim@intezer.com
```

Comment:
thank you for updating the authors throughout the rules - we really like being able to recognize the hard work of you and other contributors!